docs/whats-new/preview-5.md

```diff
@@ -0,0 +1,667 @@
+
+| Extension method                     | Package                                    |
+|--------------------------------------|--------------------------------------------|
+| `AddBicepTemplate(...)`              | `Aspire.Hosting.Azure` (unchanged)         |
+| `AddBicepTemplateString(...)`        | `Aspire.Hosting.Azure` (unchanged)         |
+| `AddAzureConstruct(...)`             | `Aspire.Hosting.Azure` (unchanged)         |
+| `AddAzureAppConfiguration(...)`      | `Aspire.Hosting.Azure.AppConfiguration`    |
+| `AddAzureApplicationInsights(...)`   | `Aspire.Hosting.Azure.ApplicationInsights` |
+| `AddAzureOpenAI(...)`                | `Aspire.Hosting.Azure.CognitiveServices`   |
+| `AddAzureCosmosDB(...)`              | `Aspire.Hosting.Azure.CosmosDB`            |
+| `AddAzureEventHubs(...)`             | `Aspire.Hosting.Azure.EventHubs`           |
+| `AddAzureKeyVault(...)`              | `Aspire.Hosting.Azure.KeyVault`            |
+| `AddAzureLogAnalyticsWorkspace(...)` | `Aspire.Hosting.Azure.OperationalInsights` |
+| `AsAzurePostgresFlexibleServer(...)` | `Aspire.Hosting.Azure.PostgreSQL`          |
+| `AsAzureRedis(...)`                  | `Aspire.Hosting.Azure.Redis`               |
+| `AddAzureSearch(...)`                | `Aspire.Hosting.Azure.Search`              |
+| `AddAzureServiceBus(...)`            | `Aspire.Hosting.Azure.ServiceBus`          |
+| `AddAzureSignalR(...)`               | `Aspire.Hosting.Azure.SignalR`             |
+| `AsAzureSqlDatabase(...)`            | `Aspire.Hosting.Azure.Sql`                 |
+| `AddAzureStorage(...)`               | `Aspire.Hosting.Azure.Storage`             |
+
+For more information, see [Azure-specific resource types](../deployment/manifest-format.md#azure-specific-resource-types).
+
+## Application model changes
+
+At the core of .NET Aspire's capabilities is the application model. The application model is a set of abstractions that allow you to define the components of your application and how they interact with each other. There were several changes to the application model in preview 5.
+
+### Allow unsecure transport for HTTP endpoints
+
+In preview 5, the **app host will crash** if an `applicationUrl` is configured with an unsecure transport. This can be avoided by setting the `ASPIRE_ALLOW_UNSECURED_TRANSPORT` environment variable. For more information, see [Allow unsecure transport in .NET Aspire](../troubleshooting/allow-unsecure-transport.md).
+
+### Enable forwarded headers by default for .NET projects
+
+If a .NET project is added to your distributed application model, and that project has endpoints defined the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is automatically set, which has the effect of enabling handling for forwarded headers. This is because the primary deployment target for Aspire applications is containerized environments where a reverse proxy is deployed in front of the workload.
```

Comment:
```suggestion
If a .NET project is added to your distributed application model, and that project has endpoints defined the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is automatically set, which has the effect of enabling handling for forwarded headers. This is because the primary deployment target for Aspire applications is containerized environments where a reverse proxy is deployed in front of the workload.

This can be disabled using the `DisableForwardedHeaders()` extension method.
```